02-peer-protocol.md

```diff
@@ -856,6 +864,11 @@ seconds, and the protocol only supports an expiry in blocks.
 specification; larger amounts are not necessary, nor wise, during the
 bootstrap phase of the network.
 
+The node _responsible_ for paying the Bitcoin fee has to maintain a small
+"additional reserve" on top of its reserve to accommodate a fee increase,
+otherwise the channel may end up in an unusable state
```

Comment:
Explain how this fix the issue: "Balance may reach the state, where any committed HTLC can't be added from the fundee-side without decreasing funder balance under the security reserve to pay fees for commitment tx. By keeping a _fee-variance buffer_, funder guarantee to the fundee at least one HTLC to rebalance channel."

Reply:
Done.